### PR TITLE
Fix/time out argument

### PR DESCRIPTION
**Pull request by hbierlee**

### CODE REVIEW

**Comment by Dekker1** [comment]
It seems this supersedes #37 and #38

**Comment by hbierlee** [comment]
Yes, I accidentally didn't make the PR's independant (although I explicilty tried to do this). Shall I force-push fix this?

File: pyproject.toml [hunk 1]
```diff
@@ -22,13 +22,13 @@ click = "^8.1"
...
 bokeh = { version =  "^3", optional = true }
+pytest = { version = "^7.4.0", optional = true }
 
 [tool.poetry.extras]
-scripts = ["tabulate", "pandas"]
+scripts = ["tabulate", "pandas", "pytest"]
 plotting = ["bokeh"]
 
 [tool.poetry.group.dev.dependencies]
-pytest = "^7.4.0"
```
Comment (Dekker1) [inline pyproject.toml:31]: I don't understand why it is no longer in the `dev` dependencies. We now have testing for the package right, so it probably belongs here still

File: src/mzn_bench/analysis/report_status.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import csv
 from pathlib import Path
 
+from typing import Iterable, Optional
```
Comment (Dekker1) [inline src/mzn_bench/analysis/report_status.py]: Was `Optional` already used somewhere? I don't see it in the changes lines

File: src/mzn_bench/analysis/report_status.py [hunk 3]
```diff
@@ -13,25 +14,14 @@ def status_from_str(s: str) -> Status:
 
 
 def report_status(
-    per_model: bool, per_problem: bool, statistics: Path, avg: str, tablefmt: str
+    keys: Iterable[str], statistics: Path, avg: str, tablefmt: str
```
Comment (Dekker1) [inline src/mzn_bench/analysis/report_status.py:17]: I would lean to a `Enum` type argument, but I suppose with this package `str` is probably fine

**Review by Dekker1 — CHANGES_REQUESTED** [review]
Could you have a quick look at the comments. Once they're resolved (probably ignore the `Enum` one), then I think this can be merged

**Comment by hbierlee** [comment]
Processed requested changes!

**Review by Dekker1 — APPROVED** [review]

**Pull request merged**